curiouslearning/FeedTheMonsterJS
# feat<FM-256>: Create Reusable Button for Level Balloons -> Implemented reusable buttons for rendering balloons in level selection and refactor code paths especially level selection scene.

Feat:  Create Reusable Button for Level Balloons

# Changes

- Deleted old component/level-config.ts.
- Create a new level-config.ts that includes handling of images and special images depending on the level and moved necessary values for level-config from level-selection-scene.ts
- Created a reusable button for levels called component/buttons/level-bloon-button.ts.
- Optimized the rendering of button levels by moving all logic to component/buttons/level-bloon-button.ts
- Code clean up for composition folder.

# How to test
- Run FTM
- Click Play on start scene.
- In level-selection-scene. button should have the visual effect when  pressed down and all levels should properly render including the stars and locks as well as load the correct game level.



Ref: [JIRA-CODE](https://curiouslearning.atlassian.net/browse/FM-256)
